source/developer/developer-setup.md

```diff
@@ -1,239 +0,0 @@
-Developer Machine Setup
------------------------------
```

Comment:
Instead of deleting the file, should we add a redirect instead, similar to our old upgrade guide?

https://docs.mattermost.com/administration/upgrade-guide.html

Worry is that we might have linked to .../developer-setup.html elsewhere, so deleting the file will break those links.